fxola/EPIC-mail
# #164650815 Setup swagger docs and implement feedback

#### What does this PR do?
- Implement API documentation Using swagger and some feedback


#### Description of Task to be completed?
- Add extra user validation
- Adjust HTTP status codes
- Implement JsDoc convention on server files
- Adjust tests to pass
- install swaggerUI package and setup API docs using swagger

#### How should this be manually tested?
Visit https://epicmail-api.herokuapp.com/api/v1/api-docs to check out the documentation

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
- PT Story link - [#164650815 ] (https://www.pivotaltracker.com/n/projects/2314937/stories/164650815 )

#### Screenshots (if appropriate)
- N/A

#### Questions:
- N/A
